test/strategy.test.js

```diff
@@ -154,6 +154,55 @@ describe('auth0 strategy', function () {
       var extraParams = this.strategy.authorizationParams({});
       should.not.exist(extraParams.acr_values);
     });
+
+    it('should map the nonce field when authParams set on strategy', function () {
+      this.strategy.authParams = { nonce: 'foo'};
+      var extraParams = this.strategy.authorizationParams({});
+      extraParams.nonce.should.eql('foo');
+    });
+
+    it('should not map the nonce field if its not a string', function () {
+      var extraParams = this.strategy.authorizationParams({nonce: 1});
+      should.not.exist(extraParams.nonce);
+    });
+
+    it('should not map the nonce field when not specified in options', function () {
+      var extraParams = this.strategy.authorizationParams({});
+      should.not.exist(extraParams.nonce);
+    });
+
+    it('should map the leeway field when set in strategy options', function () {
```

Comment:
Recommend using a different value than the default one. Impossible to test that it's using the custom one and not the modified one.

Reply:
Brilliant catch  😂